src/box/memtx_tx.h

```diff
@@ -277,10 +277,12 @@ memtx_tx_history_prepare_stmt(struct txn_stmt *stmt);
  * Make the statement's changes permanent. It becomes visible to all.
  *
  * @param stmt current statement.
+ * @param compressed_tuples the change of count of compressed tuples.
  * @return the change in space bsize.
```

Comment:
Nit: now, that there are two adjusted parameters, it's better to make them both function arguments:

```C
void
memtx_tx_history_commit_stmt(struct txn_stmt *stmt, ssize_t *bsize, int64_t *compressed_tuples);
```

Reply:
fixed